reporter/src/funTest/kotlin/reporters/CycloneDxReporterTest.kt

```diff
@@ -42,5 +49,54 @@ class CycloneDxReporterTest : WordSpec({
 
             bomParser.validate(bomFile) should beEmpty()
         }
+
+        "match the result from the official Gradle plugin" {
+            val ortResultFile = File("src/funTest/assets/gradle-all-dependencies-result.yml")
+            val bomBytesFromReporter = ByteArrayOutputStream().also { outputStream ->
+                CycloneDxReporter().generateReport(
+                    outputStream,
+                    yamlMapper.readValue(
+                        patchExpectedResult(
+                            ortResultFile,
+                            url = "https://github.com/heremaps/oss-review-toolkit.git",
+                            urlProcessed = "https://github.com/heremaps/oss-review-toolkit.git",
+                            revision = "9fded2ad79d07ab5cda44f2549301669ea10442a"
+                        ),
+                        OrtResult::class.java
+                    )
+                )
+            }.toByteArray()
+            val bomFromReporter = BomParser().parse(bomBytesFromReporter).apply { components.sortBy { it.name } }
+
+            // The file generated by the official Gradle plugin was modified in the following aspect to be comparable:
+            // - all hashes except SHA-1 were removed (because the ORT analyzer does not calculate missing hashes)
+            // - all qualifiers from the purl were removed (because the ORT analyzer does not currently handle package
+            //   manager specific qualifiers)
+            val bomFileFromPlugin = File("src/funTest/assets/gradle-all-dependencies-expected-cyclonedx-bom.xml")
+            val bomFromPlugin = BomParser().parse(bomFileFromPlugin).apply { components.sortBy { it.name } }
+
+            // TODO: Remove this once the official Gradle plugin supports getting a component's licenses, see
```

Comment:
Nit: license

Reply:
You mean singular instead of plural? Actually I deliberately used plural because Cyclone's method is called `setLicenses` and we also support multiple strings as declared licenses.